### DISCUSSION TITLE
access denied for user error

### DISCUSSION THREAD

**Discussion by Jonathon-WDB** [opendcim/openDCIM#1558] [Q&A] [answered]
I followed https://github.com/opendcim/openDCIM/wiki/Ubuntu instructions for Ubuntu 22.04 and when attempting to login to the page at the end via https://localhost/ and login I get an error showing "Error! SQLSTATE[HY000] [1045] Access denied for user 'dcim'@'localhost' (using password: YES). 
The user I created in mariadb is labrat and that is the login I used so I'm not sure why the error is for user dcim. I have been looking this up for a few days now and haven't been able to find how to get past it. Is there some step that is not listed in the instructions or am I just missing something?
Apologies if it's a simple solution as I'm not a software guy, so much of this is new to me. Please let me know if there are logs I can provide to help troubleshoot.

**Comment by wilpig** [comment, marked as answer]
That's because you deviated from the instructions.

https://github.com/opendcim/openDCIM/wiki/Ubuntu#create-database

This step had you make a user dcim. and the db.inc.php has that default user hard coded.
